magefile.go

```diff
@@ -25,6 +25,12 @@ var cliExe = "cartridge"
 var goPackageName = "github.com/tarantool/cartridge-cli/cli"
 var packagePath = "./cli"
 
+var generateModePath = filepath.Join(packagePath, "create", "codegen", "generate_mode.go")
+var generatedFilesPath = fmt.Sprintf("./%s", filepath.Join(packagePath, "create", "codegen", "static"))
```

Comment:
`packagePath` already has `./` at the beginning, is it removed on `filepath.Join`?

Reply:
yes